exercises/spiral-matrix/example.rs

```diff
@@ -0,0 +1,42 @@
+pub fn get_spiral(size: usize) -> Vec<Vec<usize>> {
+    let mut matrix: Vec<Vec<usize>> = Vec::with_capacity(size);
+    for i in 0..size {
+        matrix.push(Vec::with_capacity(size));
+        for _ in 0..size {
+            matrix[i].push(0);
+        }
+    }
+
+    let num_concentric_squares: usize = (size as f64 / 2.0).round() as usize;
```

Comment:
Trailing decimals here will always be either `.0` or `.5`. I had to think for a bit to remember what `round` did with `.5`. Using [`ceil`](https://doc.rust-lang.org/std/primitive.f64.html#method.ceil) would be a bit more explicit.